integration_tests/e2e/add.cy.ts

```diff
@@ -49,5 +49,17 @@ context('Add non-association page', () => {
     addDetailsPage.saveButton.click()
 
     Page.verifyOnPage(AddConfirmationPage)
+
+    // Clicking on the link sends a GA event
+    cy.trackGoogleAnalyticsCalls().then(googleAnalyticsTracker => {
+      cy.get('a[data-ga-category]').click()
+      cy.then(() => {
```

Comment:
Chain on the `.click()` seems to work but linter complains and also Cypress docs said it could be a problem (no idea why, possibly to do with timing?).

Anyway, this works so whatever.